src/status_im2/contexts/wallet/events.cljs

```diff
@@ -1,30 +1,101 @@
 (ns status-im2.contexts.wallet.events
   (:require
     [native-module.core :as native-module]
+    [quo.foundations.colors :as colors]
     [status-im2.common.data-store.wallet :as data-store]
     [taoensso.timbre :as log]
+    [utils.i18n :as i18n]
     [utils.re-frame :as rf]
     [utils.security.core :as security]))
 
-(rf/defn get-wallet-token
-  {:events [:wallet/get-wallet-token]}
-  [{:keys [db]}]
-  (let [params (map :address (:profile/wallet-accounts db))]
-    {:json-rpc/call [{:method     "wallet_getWalletToken"
-                      :params     [params]
-                      :on-success #(rf/dispatch [:wallet/get-wallet-token-success %])
-                      :on-error   (fn [error]
-                                    (log/info "failed to get wallet token"
-                                              {:event  :wallet/get-wallet-token
-                                               :error  error
-                                               :params params}))}]}))
-
-(rf/defn get-wallet-token-success
-  {:events [:wallet/get-wallet-token-success]}
-  [{:keys [db]} data]
-  {:db (assoc db
-              :wallet/tokens          data
-              :wallet/tokens-loading? false)})
+(rf/reg-event-fx :wallet/show-account-created-toast
+ (fn [{:keys [db]} [address]]
+   (let [{:keys [name]} (get-in db [:wallet :accounts address])]
+     {:db (update db :wallet dissoc :navigate-to-account :new-account?)
+      :fx [[:dispatch
+            [:toasts/upsert
+             {:id         :new-wallet-account-created
+              :icon       :i/correct
+              :icon-color colors/success-50
```

Comment:
can you instead use `(colors/resolve-color :success theme)` - really this needs to be handled within the toast 😢 but certainly beyond this pr

Reply:
You are right. It should be handled by the toast itself (missing the Figma 1-1 property `type`). The screens or events don't need to specify `icon-color` for toast all the time unless it's necessary. 

<img width="600" src="https://github.com/status-im/status-mobile/assets/19339952/d546e54a-549b-4c2e-b765-9de7db4e40cf">

I guess we need to do another update for the toast component 😆.
Shall we use this existing issue: https://github.com/status-im/status-mobile/issues/16733?

Reply:
Yeah let's use the existing issue. Actually there's a refactors doc. I will link that issue there too 👌